improver_tests/psychrometric_calculations/test_WetBulbTemperature.py

```diff
@@ -40,119 +40,113 @@
 
 from improver.psychrometric_calculations.psychrometric_calculations import (
     WetBulbTemperature)
-from improver.utilities.warnings_handler import ManageWarnings
 
 from ..set_up_test_cubes import set_up_variable_cube
 
 
-class Test_WetBulbTemperature(IrisTest):
-    """Test class for the WetBulbTemperature tests, setting up cubes."""
-
-    def setUp(self):
-        """Set up the initial conditions for tests."""
-        data = np.array([[185.0, 260.65, 338.15]], dtype=np.float32)
-        self.temperature = set_up_variable_cube(data)
-        data = np.array([[60., 70., 80.]], dtype=np.float32)
-        self.relative_humidity = set_up_variable_cube(
-            data, name='relative_humidity', units='%')
-        data = np.array([[1.E5, 9.9E4, 9.8E4]], dtype=np.float32)
-        self.pressure = set_up_variable_cube(
-            data, name='air_pressure', units='Pa')
-        data = np.array([[0.1, 0.2, 0.3]], dtype=np.float32)
-        self.mixing_ratio = set_up_variable_cube(
-            data, name='humidity_mixing_ratio', units='1')
-
-
-class Test_psychrometric_variables(Test_WetBulbTemperature):
+class Test_psychrometric_variables(IrisTest):
     """Test calculations of one-line variables: svp in air, latent heat,
     mixing ratios, etc"""
 
+    def setUp(self):
+        """Set up shared input data"""
+        self.mixing_ratio = np.array([0.1, 0.2, 0.3], dtype=np.float32)
+        self.specific_heat = np.array(
+            [1089.5, 1174., 1258.5], dtype=np.float32)
+        self.latent_heat = np.array(
+            [2531771., 2508371., 2484971.], dtype=np.float32)
+        self.temperature = np.array([185.0, 260.65, 338.15], dtype=np.float32)
+
     def test_calculate_latent_heat(self):
         """Test latent heat calculation"""
-        expected = [[2707271., 2530250., 2348900.]]
+        expected = [2707271., 2530250., 2348900.]
         result = WetBulbTemperature()._calculate_latent_heat(self.temperature)
         self.assertArrayAlmostEqual(result.data, expected)
 
     def test_calculate_mixing_ratio(self):
         """Test mixing ratio calculation"""
-        expected = [[6.06744631e-08, 1.31079322e-03, 1.77063149e-01]]
+        pressure = np.array([1.E5, 9.9E4, 9.8E4], dtype=np.float32)
+        expected = [6.06744631e-08, 1.31079322e-03, 1.77063149e-01]
```

Comment:
Given the default behaviour of `assertArrayAlmostEqual` of `decimal=6`, I don't think the first value is being checked correctly. There also doesn't seem to be an appropriate setting for this whole array. Perhaps this one needs checking individually.